src/pages/login/NusLogin.tsx

```diff
@@ -0,0 +1,87 @@
+import { Button, ButtonGroup, Card, Divider, Elevation, H1, H3 } from '@blueprintjs/core';
+import classNames from 'classnames';
+import React, { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
+import SessionActions from 'src/commons/application/actions/SessionActions';
+import Constants from 'src/commons/utils/Constants';
+import classes from 'src/pages/login/NusLogin.module.scss';
+
+import sourceAcademyLogo from '../../assets/SA.jpg';
+
+const nusProviders = [...Constants.authProviders.entries()]
+  .map(([id, { name }]) => ({
+    id,
+    name
+  }))
+  .filter(e => e.name.includes('NUS'));
+
+const NUS_LOGO =
```

Comment:
This should ideally be in its own asset file instead of being hardcoded

Reply:
It's only used in one page, that's why I thought no need to put in an asset. Also, it's NUS-specific, not Source Academy, so it felt weird to put under Source Academy assets.